2-projects/1-todo-list/index.html

```diff
@@ -61,12 +42,20 @@
                     />
                   </div>
                   <div class="col-auto">
-                    <button
+                    <button 
+                      id="add-todo-btn"
                       onclick="addNewTodo(event)"
                       class="btn btn-primary mb-3"
                     >
                       Add Todo
                     </button>
+                    <button 
+                      id="delete-all-completed-todo-btn"
+                      onclick="deleteAllCompletedTodos(event)"
```

Comment:
well done for adding a JS function to an html button. Another way to do this would be to add a click event listener to this button using its ID in script.js

Reply:
Thanks Ellie, I wanted to make it similar to do other onclick button.